todoapp/app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TasksDaoTest.kt

```diff
@@ -39,124 +39,139 @@ import org.junit.runner.RunWith
                 ToDoDatabase::class.java).build()
     }
 
-    @After fun closeDb() {
-        database.close()
-    }
+    @After fun closeDb() = database.close()
+
 
     @Test fun insertTaskAndGetById() {
         // When inserting a task
-        database.taskDao().insertTask(TASK)
+        database.taskDao().insertTask(DEFAULT_TASK)
 
         // When getting the task by id from the database
-        val loaded = database.taskDao().getTaskById(TASK.id)
+        val loaded = database.taskDao().getTaskById(DEFAULT_TASK.id)
 
         // The loaded data contains the expected values
-        assertTask(loaded, "id", "title", "description", true)
+        assertTask(loaded, DEFAULT_ID, DEFAULT_TITLE, DEFAULT_DESCRIPTION, DEFAULT_IS_COMPLETED)
     }
 
     @Test fun insertTaskReplacesOnConflict() {
-        //Given that a task is inserted
-        database.taskDao().insertTask(TASK)
+        // Given that a task is inserted
+        database.taskDao().insertTask(DEFAULT_TASK)
 
+        val newTitle = "title2"
+        val newDescription = "description2"
+        val newId = "id"
+        val newIsCompleted = true
         // When a task with the same id is inserted
-        val newTask = Task("title2", "description2", "id").apply {
-            isCompleted = true
+        val newTask = Task(newTitle, newDescription, newId).apply {
```

Comment:
nit: could be a constant, too, since it's being reused in `updateTaskAndGetById`

Reply:
Done.